src/main/java/core/basesyntax/exeptions/NotEnoughFruitsException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exeptions;
+
+public class NotEnoughFruitsException extends Exception {
+    public NotEnoughFruitsException(String message) {
```

Comment:
```suggestion
    public NotEnoughFruitsException(String message, Throwable cause) {
```

for all your custom exeptions - include the original exception that was a reason of the custom one
also, let's maybe make them extend RuntimeException?

Reply:
а що має робити Throwable cause?

Reply:
discussed